src/app/auth.guard.ts

```diff
@@ -3,18 +3,24 @@ import { Injectable } from "@angular/core";
 import { ActivatedRouteSnapshot, CanActivate, Router, RouterStateSnapshot, UrlTree } from "@angular/router";
 
 import { AuthenticationService } from "./shared/services/authentication.service";
+import { Observable, map } from "rxjs";
```

Comment:
Please move the import from rxjs to the general import section

Reply:
done